### PR TITLE
Add additional note fields

### PR DESCRIPTION
**Pull request by warrd**
https://github.com/wellcometrust/platform/issues/4114

### CODE REVIEW

File: common/display/src/main/scala/uk/ac/wellcome/display/models/DisplayNote.scala [hunk 1]
```diff
@@ -68,5 +68,19 @@ object DisplayNoteType {
         DisplayNoteType("location-of-original", "Location of original")
       case BindingInformation(_) =>
         DisplayNoteType("binding-detail", "Binding detail")
+      case BiographicalNote(_) =>
```
Comment (jamesgorrie) [inline common/display/src/main/scala/uk/ac/wellcome/display/models/DisplayNote.scala]: What's the difference between this and `BibliographicalInformation `?

Comment (warrd) [reply common/display/src/main/scala/uk/ac/wellcome/display/models/DisplayNote.scala]: bibliographical vs biographical

Comment (jamesgorrie) [reply common/display/src/main/scala/uk/ac/wellcome/display/models/DisplayNote.scala]: Sorry, Christmas laziness has come early!

Comment (warrd) [reply common/display/src/main/scala/uk/ac/wellcome/display/models/DisplayNote.scala]: :christmas_tree:

**Pull request merged**